nyx_client/nyx_client/client.py

```diff
@@ -381,7 +383,7 @@ def _data_from_response_object(self, obj: dict[str, Any]) -> Data:
             org=self.org,
             url=obj["accessURL"],
             content_type=obj["contentType"],
-            creator=obj["creator"],
+            creator=obj.get("creator", self.org),
             categories=obj["categories"],
             genre=obj["genre"],
             size=obj["size"],
```

Comment:
Should `connection_id` also be set here, if present? (Would make sense for local products)

I just realised that we probably don't want to return connection ID for all the remote products even though it is "public" metadata :thinking: